### DISCUSSION TITLE
希望可以增加交换 A/B X/Y 按键的选项

### DISCUSSION THREAD

**Discussion by zzyyuanmi** [xfangfang/wiliwili#411] [Ideas]
NS和PC双端用户，然而Xbox的键位与NS是反的，用的时候经常按错。是否可以在设置界面增加一个选项，用于交换这两对按键

**Comment by dragonflylee** [comment]
可以将 wiliwili 添加到 steam 运行，参考 https://www.bilibili.com/read/cv7986320/

**Reply by zzyyuanmi** [reply]
通过 steam 运行主要是想在 PC 上用 NS Pro 手柄吧，我的意思是在 PC 上用 Xbox 键位手柄的时候，操作按键会和 NS 端相反。虽然确实也能通过 steam 重新映射按键，但还是希望可以原生支持一下，通过 steam 启动还是麻烦了些

**Comment by xfangfang** [comment]
这个我明天实现一下吧，切换按键之后，我希望键盘的 enter/esc 不切换，需要改一些代码和重做一个键盘映射的字体。

**Comment by xfangfang** [comment]
@zzyyuanmi 支持了 A-B X-Y 交换的配置选项，见：https://github.com/xfangfang/wiliwili/wiki#交换手柄按键

下载：https://github.com/xfangfang/wiliwili/actions/runs/11398005561

---

更新：应用设置内，添加了前端选项

**Reply by zzyyuanmi** [reply]
用上了，感谢❤️
